src/sparseml/exporters/transforms/fold_conv_div_bn.py

```diff
@@ -0,0 +1,100 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy
+from onnx import ModelProto, numpy_helper
+
+from sparseml.exporters.transforms.onnx_transform import OnnxTransform
+from sparseml.exporters.transforms.utils.matching import (
+    MatchResult,
+    get_structural_matches,
+)
+from sparseml.onnx.utils import (
+    ONNXGraph,
+    get_batch_norm_params,
+    get_init_by_name,
+    remove_node_and_params_from_graph,
+)
+
+
+__all__ = ["FoldConvDivBn"]
+
+
+class FoldConvDivBn(OnnxTransform):
+    """
+    Transforms
```

Comment:
need to update docstring here to state purpose of transform (ie that this specifically gets rid of the undo step of the fake batch norm fold that happens in pytorch qat)... confusing even for the initialized...